csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs

```diff
@@ -178,6 +178,9 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
         /// <returns>The converted name.</returns>
         private static string OperatorSymbol(Context cx, IMethodSymbol method)
         {
+            if (method.ExplicitInterfaceImplementations.Any())
+                return OperatorSymbol(cx, method.ExplicitInterfaceImplementations.First());
```

Comment:
Why is this change needed?

Reply:
It has been an issue all along, but with the upgrade we ran into it in the tests, which we didn't before.